lighting/material/new.glsl

```diff
@@ -43,6 +43,8 @@ void materialNew(out Material _mat) {
     // Surface data
     _mat.position           = (SURFACE_POSITION).xyz;
     _mat.normal             = materialNormal();
+    _mat.sdf                = 0.0;
```

Comment:
Maybe:

```
_mat.sdf = RAYMARCH_MAX_DIST;
```

So it doesn't need to be define on `raymarchMap()`.

This is bringing a pattern that rendering method needs to be define first. Same for `pbrClearCoat()` We should need to add this on some documentation

Reply:
@patriciogonzalezvivo  That's also a good idea, but subject to the same circular dependency issues. Shall we park it for now?